hackathon_site/event/jinja2/event/landing.html

```diff
@@ -250,18 +250,18 @@ <h3 class="center white-text">FAQ</h3>
                 <!-- Bronze -->
 
                 <!-- Partners offset-m3 offset-s1-->
-<!--                <a href="https://www.ecobee.com/en-ca/" class="col m2 s3 offset-s1" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/ecobee_white.svg') }}" alt="ecobee" /></a>-->
+                <a href="https://mlh.io/" class="col m2 s3 offset-s1" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/mlh-logo-color.svg') }}" alt="ecobee" /></a>-->
 <!--                <a href="https://www.amd.com/" class="col m2 s3" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/AMD_white.svg') }}" alt="AMD" /></a>-->
 <!--                <a href="https://hatchery.engineering.utoronto.ca" class="col m2 s3" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/The_Hatchery_white.svg') }}" alt="The Hatchery" /></a>-->
 <!--                <a href="https://www.voiceflow.com" class="col m2 s3" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/Voiceflow_white.svg') }}" alt="Voiceflow" /></a>-->
 <!--                <a href="https://www.huawei.com/ca/" class="col m2 s3" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/Huawei_white.svg') }}" alt="Huawei" /></a>-->
-<!--            </div>-->
-<!--            <p class="center white-text">-->
-<!--                If you'd like to sponsor us, feel free to reach out:-->
-<!--                <a href="mailto:newhacks@ieee.utoronto.ca" class="hoverLink lightText">newhacks@ieee.utoronto.ca</a>-->
-<!--            </p>-->
-<!--        </div>-->
-<!--    </div>-->
+            </div>-->
+            <p class="center white-text">-->
+                If you'd like to sponsor us, feel free to reach out:-->
+                <a href="mailto:newhacks@ieee.utoronto.ca" class="hoverLink lightText">newhacks@ieee.utoronto.ca</a>-->
+            </p>-->
+        </div>-->
+    </div>-->
 
```

Comment:
Somehow, there's a closing div tag missing from the `ombreBackgroundDark2` div on line 235. Add a closing div tag here